recipes/mosquitto/2.x/test_package/basic-1.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * This example shows how to publish messages from outside of the Mosquitto network loop.
+ */
+
+#include <mosquitto.h>
```

Comment:
Please don't keep the word "example" we keep having issue with people think the "test_package" is an example

Please save us from extra issues!

```suggestion
#include <mosquitto.h>
```